src/shared/components/TextArea/TextArea.style.ts

```diff
@@ -0,0 +1,61 @@
+import { colors, sizes, transitions, typography } from '@/shared/theme'
+import styled from '@emotion/styled'
+
+export const TextAreaWrapper = styled.div`
+  max-width: 300px;
+`
+
+export const StyledTextArea = styled.textarea`
+  background: ${colors.black};
```

Comment:
Looked into the design and turns out it has to be transparent. Look here for example: https://www.figma.com/file/rBjfm2tJ2Pr9M0UNTduU9v/Joystream-Atlas?node-id=3079%3A18143